diff --unified --recursive --text AirSim-1.8.1/cmake/AirLib/CMakeLists.txt AirSim-1.8.1.new/cmake/AirLib/CMakeLists.txt
--- AirSim-1.8.1/cmake/AirLib/CMakeLists.txt	2022-07-17 16:37:48.000000000 +0300
+++ AirSim-1.8.1.new/cmake/AirLib/CMakeLists.txt	2022-12-20 21:05:32.451149321 +0300
@@ -28,6 +28,7 @@
 CommonTargetLink()
 target_link_libraries(${PROJECT_NAME} ${RPC_LIB})
 target_link_libraries(${PROJECT_NAME} MavLinkCom)
+target_link_libraries(${PROJECT_NAME} Eigen3::Eigen)
 
 #string(SUBSTRING ${CMAKE_STATIC_LINKER_FLAGS} 9 -1 "BUILD_PLATFORM")
 #find_package(Threads REQUIRED)
diff --unified --recursive --text AirSim-1.8.1/cmake/CMakeLists.txt AirSim-1.8.1.new/cmake/CMakeLists.txt
--- AirSim-1.8.1/cmake/CMakeLists.txt	2022-07-17 16:37:48.000000000 +0300
+++ AirSim-1.8.1.new/cmake/CMakeLists.txt	2022-12-20 21:05:18.002759755 +0300
@@ -1,10 +1,8 @@
 ﻿cmake_minimum_required(VERSION 3.5.0)
 project(AirSim)
 
-add_subdirectory("rpclib_wrapper")
 add_subdirectory("AirLib")
 add_subdirectory("MavLinkCom")
-add_subdirectory("AirLibUnitTests")
 add_subdirectory("HelloDrone")
 add_subdirectory("HelloSpawnedDrones")
 add_subdirectory("HelloCar")
diff --unified --recursive --text AirSim-1.8.1/cmake/cmake-modules/CommonSetup.cmake AirSim-1.8.1.new/cmake/cmake-modules/CommonSetup.cmake
--- AirSim-1.8.1/cmake/cmake-modules/CommonSetup.cmake	2022-07-17 16:37:48.000000000 +0300
+++ AirSim-1.8.1.new/cmake/cmake-modules/CommonSetup.cmake	2022-12-20 21:05:59.464874195 +0300
@@ -6,7 +6,8 @@
 endmacro(CommonTargetLink)
 
 macro(IncludeEigen)
-    include_directories(${AIRSIM_ROOT}/AirLib/deps/eigen3)
+    find_package(Eigen3 REQUIRED)
+    include_directories(${Eigen_INCLUDE_DIRS})
 endmacro(IncludeEigen)
 
 macro(AddExecutableSource)
